### PR TITLE
fix: golangci-lint error

### PR DESCRIPTION
**Pull request by FrankYang0529**
#### Which issue(s) this PR fixes:
https://github.com/longhorn/longhorn/issues/7448

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -24,14 +24,14 @@ RUN apt-get update && \
 RUN rm -f /bin/sh && ln -s /bin/bash /bin/sh
 
 # Install Go & tools
-ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
+ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm64=arm64 GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
-RUN wget -O - https://storage.googleapis.com/golang/go1.14.1.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local
+RUN wget -O - https://storage.googleapis.com/golang/go1.21.3.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local
```
Comment (derekbit) [inline Dockerfile.dapper:29]: go.mod is using 1.22.0([ref](https://github.com/longhorn/go-iscsi-helper/blob/master/go.mod#L3)). Should we use go1.22.0 here?

File: Dockerfile.dapper [hunk 2]
```diff
@@ -24,14 +24,14 @@ RUN apt-get update && \
...
 RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.55.2
 
 # Docker
 ENV DOCKER_URL_amd64=https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/docker-ce_17.03.3~ce-0~ubuntu-xenial_amd64.deb \
-    DOCKER_URL_arm=https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/arm64/docker-ce_18.06.1~ce~3-0~ubuntu_arm64.deb \
+    DOCKER_URL_arm64=https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/arm64/docker-ce_18.06.1~ce~3-0~ubuntu_arm64.deb \
```
Comment (derekbit) [inline Dockerfile.dapper:34]: The docker version is too old. Can you help create a ticket for updating them? Thank you.

Comment (FrankYang0529) [reply Dockerfile.dapper:34]: https://github.com/longhorn/longhorn/issues/8449

File: Dockerfile.dapper (hunk 1 above)
Comment (FrankYang0529) [reply Dockerfile.dapper]: Updated it. Thank you.

**Review by derekbit — DISMISSED** [review]

**Comment by derekbit** [comment]
@FrankYang0529 Need to resolve the conflict. Thank you.

**Review by derekbit — APPROVED** [review]

**Pull request merged**